pandas/tests/formats/test_style.py

```diff
@@ -348,6 +348,109 @@ def test_bar_0points(self):
                              ', transparent 0%)']}
         self.assertEqual(result, expected)
 
+    def test_bar_align_zero_pos_and_neg(self):
+        df = pd.DataFrame({'A': [-10, 0, 20, 90]})
```

Comment:
You mean this pull request number? https://github.com/pandas-dev/pandas/pull/14757?